dojo_plugin/api/v1/docker.py

```diff
@@ -349,6 +349,108 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+@docker_namespace.route("/module-challenges")
+class ModuleChallenges(Resource):
```

Comment:
TODO: Move this to [api/v1/docker.py](https://github.com/pwncollege/dojo/pull/926/files#diff-60049d9f172fa6fb627b18f0ee5df714accf9ee6430e728a23152d8d28e22dea)